app/src/main/java/com/mathewsachin/fategrandautomata/accessibility/ScriptRunnerService.kt

```diff
@@ -213,9 +226,19 @@ class ScriptRunnerService : AccessibilityService() {
     override fun onServiceConnected() {
         Timber.info { "Accessibility Service bound to system" }
 
+        // We only want events from FGO
+        serviceInfo = serviceInfo.apply {
```

Comment:
So you're setting the package names when the accessibility service starts so you don't have to edit the script_runner_service.xml on package name changes, right?

Reply:
Yes